cmd/system-probe/config/adjust_usm.go

```diff
@@ -20,6 +20,7 @@ func adjustUSM(cfg model.Config) {
 	if cfg.GetBool(smNS("enabled")) {
 		applyDefault(cfg, netNS("enable_http_monitoring"), true)
 		applyDefault(cfg, netNS("enable_https_monitoring"), true)
+		applyDefault(cfg, smNS("tls", "go", "enabled"), true)
```

Comment:
that's not the place to put it.
you're breaking the backward compatibility

Reply:
Could you explain? I don't get it.

Reply:
Try to run usm with the follow configuration:

```yaml
service_monitoring_config:
  enabled: true
  enable_go_tls_support: false
```

Reply:
Changed it to only apply the default if the deprecated option is not set.